code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -70,6 +70,43 @@
 				/obj/item/grenade/clusterbuster,
 				/obj/item/storage/box/syndie_kit/throwing_weapons = 3)
 
+/obj/effect/spawner/lootdrop/prison_contraband
+	name = "prison contraband loot spawner"
+	loot = list(/obj/item/clothing/mask/cigarette/space_cigarette = 4,
+				/obj/item/clothing/mask/cigarette/robust = 2,
+				/obj/item/clothing/mask/cigarette/carp = 3,
+				/obj/item/clothing/mask/cigarette/uplift = 2,
+				/obj/item/clothing/mask/cigarette/dromedary = 3,
+				/obj/item/clothing/mask/cigarette/robustgold = 1,
+				/obj/item/storage/fancy/cigarettes/cigpack_uplift = 3,
+				/obj/item/storage/fancy/cigarettes = 3,
+				/obj/item/clothing/mask/cigarette/rollie/cannabis = 4,
+				/obj/item/toy/crayon/spraycan = 2,
+				/obj/item/crowbar = 1,
+				/obj/item/assembly/flash/handheld = 1,
+				/obj/item/restraints/handcuffs/cable/zipties = 1,
+				/obj/item/restraints/handcuffs = 1,
+				/obj/item/radio/off = 1,
+				/obj/item/lighter = 3,
+				/obj/item/storage/box/matches = 3,
+				/obj/item/reagent_containers/syringe = 2,
+				/obj/item/reagent_containers/glass/bottle/morphine = 2,
+				/obj/item/reagent_containers/glass/bottle/fentanyl = 1,

```

Comment:
make this come in a syringe, not a bottle

Reply:
@Bawhoppen any comment on this?

Reply:
Oh yes, sorry for the late reply. The original thought for this and also the lighters was to add some potential scarcity, so prisoners might need to trade amongst themselves. Although, it probably would just be better to put it in a syringe directly like you suggest,